Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SavePlot1D.py

```diff
@@ -234,7 +234,7 @@ def saveImage(self):
         # get the flag to pop out canvas or not
         pop_canvas = self.getProperty('PopCanvas').value
 
-        plt.tight_layout(1.08)
+        plt.tight_layout()
```

Comment:
Why have we removed the 1.08?

Reply:
Magic numbers are bad, and it caused no issues.